src/General/Alert/Alert.tsx

```diff
@@ -130,12 +130,20 @@ const Alert = ({
   ) : null;
 };
 
-interface Props {
-  type: string;
-  message: React.ReactNode;
+export type AlertType = 'success' | 'warning' | 'danger' | 'info';
+
+export interface Props {
+  /** Set Alert's style. */
+  type: AlertType;
```

Comment:
Since you set the default value `info` in line 25, I think this property should be optional now.